### DISCUSSION TITLE
HRRRX - HRRR Experiemental Archive

### DISCUSSION THREAD

**Discussion by msw17002** [blaylockbk/Herbie#278] [Q&A]
Hello,

Not a bug/issue, more of a question...

Where can I download HRRRX data? 

You mentioned that, "experimental HRRR (is) no longer archived on Pando." Has the HRRRX archive migrated or has it been entirely removed by IEM/Pando? 

Pando: https://home.chpc.utah.edu/~u0553130/Brian_Blaylock/cgi-bin/hrrr_download.cgi?model=hrrrX&field=sfc&date=2020-01-16&link2=metadata

I'm looking for HRRRX data in ~2020 for the CONUS.

**Comment by blaylockbk** [comment]
Hi @msw17002 

Good question. I'm not aware of any place the HRRR-X is archived publicly. The HRRR model is developed by GSL. A few suggestions:
1. You can ask the HRRR developers if the HRRR-X data is available (email the scientific contact listed at the bottom right of the page https://rapidrefresh.noaa.gov/hrrr/). 
2. Ask your question on https://github.com/NOAA-GSL/GSL-Regional-Model-Forum/discussions
3. @EricJames-NOAA may be able to provide some details.
